instrumentation/google.golang.org/grpc/otelgrpc/test/interceptor_test.go

```diff
@@ -89,7 +89,13 @@ func TestUnaryClientInterceptor(t *testing.T) {
 
 	sr := tracetest.NewSpanRecorder()
 	tp := trace.NewTracerProvider(trace.WithSpanProcessor(sr))
-	unaryInterceptor := otelgrpc.UnaryClientInterceptor(otelgrpc.WithTracerProvider(tp))
+	unaryInterceptor := otelgrpc.UnaryClientInterceptor(
+		otelgrpc.WithTracerProvider(tp),
+		otelgrpc.WithMessageEvents(otelgrpc.ReceivedEvents, otelgrpc.SentEvents),
```

Comment:
This test should also be updated so the unary client interceptor tests without events, and with some of them only.

Reply:
Updated.